### PR TITLE
add version 0.56

### PR DESCRIPTION
**Pull request by azazellochg**
This adds new version support.
New binaries: https://www.mrc-lmb.cam.ac.uk/kzhang/Gautomatch/Gautomatch_v0.56/
This closes https://github.com/I2PC/scipion/issues/1640

### CODE REVIEW

**Comment by delarosatrevin** [comment]
@azazellochg I think the binaries should be handled by the plugin itself. @pconesa, is the installation of binaries from the plugin already working?

**Comment by pconesa** [comment]
Hi, @azazellochg, @delarosatrevin  , binaries are defined by the plugin, see https://github.com/scipion-em/scipion-em-appion/blob/devel/appion/plugin.py#L22

@fonsecareyna82 just finished today to fix some issues regarding the installation and the new plugin structure. 

I think we can push changes tomorrow. We will also update the documentation related to this bit. I just roughly removed sections form Yaizas old documentation, but this is not final, but useful.

https://github.com/I2PC/scipion/wiki/Guide:-developing-scipion-plugins

**Comment by pconesa** [comment]
Binary files can also be hosted anywhere, reachable by an http download.

**Comment by azazellochg** [comment]
Binaries and pypi files added to devel branch now.

File: gautomatch/__init__.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ def getEnviron(cls):
     @classmethod
     def getProgram(cls):
         """ Return the program binary that will be used. """
-        return os.path.join(cls.getHome('bin'), cls.getVar(GAUTOMATCH))
+        return os.path.join(cls.getHome('bin'),
+                            os.path.basename(cls.getVar(GAUTOMATCH)))
```
Comment (azazellochg) [inline gautomatch/__init__.py:72]: thanks! I missed that.

**Comment by delarosatrevin** [comment]
Tests are passing and also tested in other project. Still the binary seems missing.
@azazellochg , now there are a bunch of cuda/sm/ binary versions...which one do you propose that we bundle into our binary tgz for automatic install?

**Comment by azazellochg** [comment]
Gautomatch_v0.56_sm62_cu8.0 would be good. sm* computing has backward compatibility so it will work on old cards as well.

**Pull request merged**